src/Modules/Shared/Microsoft.PowerShell.Host/Microsoft.PowerShell.Host.psd1

```diff
@@ -4,10 +4,8 @@ Author="PowerShell"
 CompanyName="Microsoft Corporation"
 Copyright="Copyright (c) Microsoft Corporation. All rights reserved."
 ModuleVersion="3.0.0.0"
```

Comment:
There's a request to update our in-package ModuleVersion https://github.com/PowerShell/PowerShell/issues/2136, I think we can just have the ModuleVersion match the PSCore version "6.1.0.0" for this release until we can actually decouple them.